ui/guest/src/util.tsx

```diff
@@ -460,6 +466,15 @@ export function retrieveProperty(object: object, prop: string): any {
       : prop.split('.').reduce((value, prop) => value[prop], object);
 }
 
+export function deleteProperty(object: object, prop: string): object {
+  return Object.keys(object).reduce((next, key) => {
+    if (key !== prop) {
+      next[key] = object[key];
+    }
+    return next;
+  }, {});
+}
```

Comment:
This is sort of equivalent to reversePluckProps. 

A deleteProperty should be more like
```typescript
export function deleteProperty<T, P extends keyof T>(object: T, prop: P): Omit<T, P> {
  delete object[prop];
  return object;
}
```

Notice it removes the prop from the source object.

Reply:
Add to app's object.ts too.